### PR TITLE
create projects page

### PR DESCRIPTION
**Pull request by thecoder876**
add a projects page and edit home page
cc @wgyt

### CODE REVIEW

File: projects.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Projects
+
```
Comment (wgyt) [inline projects.md:2]: Can you add all of the onedot projects, including 1auth, 1cdn, 1class, 1host, 1social, and 1state

**Review by wgyt — CHANGES_REQUESTED** [review]

File: projects.md (hunk 1 above)
Comment (wgyt) [reply projects.md:2]: also add the logos.
https://cdn.onedot.cf/brand/PNG/Transparent/

Comment (thecoder876) [reply projects.md:2]: > Can you add all of the onedot projects, including 1auth, 1cdn, 1class, 1host, 1social, and 1state

@wgyt ok, i only added the ones with a gh repo, so lemme add those

Comment (thecoder876) [reply projects.md:2]: > also add the logos.
> https://cdn.onedot.cf/brand/PNG/NoPadding/

@wgyt ok

File: projects.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+[Website](https://1host.js.org)
+
+## 1Cdn
+[GitHub Repo](https://github.com/onedotprojects/cdn)
+
+## 1Class
+*No info currently available*
+
+## 1Social
+*No info currently available*
```
Comment (wgyt) [inline projects.md:25]: Please link to the repo
```suggestion
[Github Repo](https://github.com/wgytwebsites/social.wgyt.tk)```

**Review by wgyt — CHANGES_REQUESTED** [review]

**Review by wgyt — APPROVED** [review]

File: projects.md (hunk 2 above)
Comment (thecoder876) [reply projects.md]: > Please link to the repo

@wgyt done

**Comment by thecoder876** [comment]
@wgyt why i didn't finish

**Comment by wgyt** [comment]
> @wgyt why i didn't finish
> i haven't added the logos yet

I guess you can make a new pr

**Pull request merged**